README.rst

```diff
@@ -92,10 +92,9 @@ the ``WATCHMAN_CHECKS`` setting. In ``settings.py``::
         'another.module.path.to.callable',
     )
 
-Checks now have the same contract as context processors: they consume a
-``request`` and return a ``dict`` whose keys are applied to the JSON response::
```

Comment:
@kezabelle I know you wrote this originally so that all checks take in a `request` argument - in practice with the checks I saw, this wasn't necessary, and caused issues with expanding the usage of watchman outside of a request/response cycle (eg. this management command change).

Did you have some custom checks you've written which are expecting to use a `request` object? If so, any thoughts on how to support both types?